k8s/templates/sumo-app.yaml.j2

```diff
@@ -0,0 +1,85 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ kubernetes.app_name }}
+    type: web
+  name: {{ kubernetes.deployment_name }}
+  namespace: {{ kubernetes.namespace }}
+spec:
+  replicas: {{ kubernetes.replicas }}
+  selector:
+    matchLabels:
+      app: {{ kubernetes.app_name }}
+      type: web
+  strategy:
+    rollingUpdate:
+      maxSurge: {{ kubernetes.rollouts.max_surge }}
+      maxUnavailable: {{ kubernetes.rollouts.max_unavailable }}
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ kubernetes.app_name }}
+        type: web
+      name: {{ kubernetes.deployment_name }}
+      namespace: {{ kubernetes.namespace }}
+    spec:
+      containers:
+      - args:
+        - {{ kubernetes.command }}
+        command:
+        - /bin/bash
+        - -c
+        env:
+            {%- for name in app|sort -%}
+              {% set value = app[name] %}
+            - name: {{ name|upper }}
+              {% if value == 'SECRET' -%}
+              valueFrom:
+                secretKeyRef:
+                  key: {{ name|replace('_', '-') }}
+                  name: "{{ kubernetes.secrets_name }}"
+              {%- else -%}
+                value: "{{ value }}"
+              {%- endif -%}
+            {%- endfor %}
+        image: {{ kubernetes.image.repo }}:{{ kubernetes.image.tag }}
+        imagePullPolicy: {{ kubernetes.image.pull_policy }}
+        name: sumo-{{ kubernetes.target_environment }}-web
+        {%- if kubernetes.readiness %}
+        readinessProbe:
+          httpGet:
+              port: {{ app.port }}
+              path: /readiness/
+              httpHeaders:
+                  - name: Host
+                    value: {{ app.allowed_hosts.split(',')[0] }}
+          failureThreshold: {{ kubernetes.readiness.failure_threshold }}
+          initialDelaySeconds: {{ kubernetes.readiness.initial_delay_seconds }}
+          periodSeconds: {{ kubernetes.readiness.period_seconds }}
+          successThreshold: {{ kubernetes.readiness.success_threshold }}
+          timeoutSeconds: {{ kubernetes.readiness.timeout_seconds }}
+        {%- endif %}
+        {%- if kubernetes.liveness %}
+        livenessProbe:
+          httpGet:
+              port: {{ app.port }}
+              path: /healthz/
+              httpHeaders:
+                  - name: Host
+                    value: {{ app.allowed_hosts.split(',')[0] }}
```

Comment:
How does this work with django-allow-cidr?

Reply:
Here we use the first hostname from allowed_hosts and django-allow-cidr respects allowed_hosts.

Reply:
> @glogiotatidis I see that MDN is running in Frankfurt and has a liveness probe defined, the pods appear to be up and running:

Used similar settings and it works now, thanks!